r/R/dplyr-functions.R

```diff
@@ -215,6 +215,52 @@ nse_funcs$nchar <- function(x, type = "chars", allowNA = FALSE, keepNA = NA) {
   }
 }
 
+nse_funcs$paste <- function(..., sep = " ", collapse = NULL, recycle0 = FALSE) {
+  assert_that(is.null(collapse))
```

Comment:
Should we do better than this (i.e. explain in the error message that Arrow doesn't support it currently)?

Reply:
Improved the message in c8a4ab2